src/components/atoms/Ticker/Tick/index.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+
+const Tick = (props: Props) => {
+  const {
+    value,
+    height,
+    range,
+    className,
+    currentClassName,
+    hiddenClassName
+  } = props;
+
+  const index = range.indexOf(value);
+
+  return (
+    <span className={`ticker42Tick ${className}`}>
+      <span
+        className={`ticker42hiddenPiece ${className}`}
```

Comment:
same className ?

Reply:
discussed already and changed in the file.